OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -66,6 +66,15 @@ public class MobileInfo : IMobileInfo, IOccupySpaceInfo, IFacingInfo, UsesInit<F
 		[Desc("Can the actor be ordered to move in to shroud?")]
 		public readonly bool MoveIntoShroud = true;
 
+		[Desc("Randomly select a sound from this list to play when this actor beings moving.")]
```

Comment:
just as a question but how does this sentence make sense?
"when this actor beings moving."